performance-tools/performance-cluster/scripts/helpers/install-kyma.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
```

Comment:
Duplicate functionality, would a more generic rewrite of the installer script help?